code/identity/src/main/java/com/adobe/marketing/mobile/identity/ConfigurationSharedStateIdentity.java

```diff
@@ -83,6 +67,33 @@ void getConfigurationProperties(final Map<String, Object> sharedState) {
                                 Defaults.DEFAULT_MOBILE_PRIVACY.getValue()));
     }
 
+    /**
+     * The Experience Cloud organization ID.
+     *
+     * @return the Experience Cloud organization ID or null if one wasn't set.
+     */
+    @Nullable String getOrgID() {
+        return orgID;
+    }
+
+    /**
+     * The privacy status.
+     *
+     * @return the privacy status; defaults to {@link Defaults#DEFAULT_MOBILE_PRIVACY}.
+     */
+    @NonNull MobilePrivacyStatus getPrivacyStatus() {
+        return privacyStatus;
+    }
+
+    /**
+     * The Experience Cloud server.
+     *
+     * @return the Experience Cloud server; defaults to {@link Defaults#SERVER}
+     */
+    @NonNull String getMarketingCloudServer() {
```

Comment:
since you did a bunch of changes in this class, can we also rename these methods and properties marketingCloudServer -> experienceCloudServer